Zadatak4_cas2.java

```diff
@@ -0,0 +1,71 @@
+package endava;
+
+import java.util.Scanner;
+
+public class Zadatak4_cas2 {
+
+	static double sabiranje(int x, int y) {

```

Comment:
U svim metodama se operacija racunanja izvrsava dva puta, jednom kada se prikazuje tekst, a drugi put kada se vraca rezultat racunanja. Dovoljno je samo jednom, pri cemu se procesorsko vreme stedi:
```suggestion
	static double sabiranje(int x, int y) {
		int rezultat = x + y;
		System.out.println(rezultat);
		return rezultat;
	}
```